index.html

```diff
@@ -0,0 +1,293 @@
+      },
+    };
+    </script>
+  </head>
+  <body data-cite="credential-management-1 WebCryptoAPI">
+    <section id="abstract">
+      <p>
+        This document standardizes an API solution for requesting digital
+        identity credentials (e.g., a drivers license) or verifying some aspect
+        of a digital identity credential (e.g., proof of age). The API builds
+        on the [[[credential-management-1]]] as a means to request a digital
+        credential from the user agent or underlying platform.
+      </p>
+    </section>
+    <section id="sotd">
+      <p>
+        This is an unofficial proposal.
+      </p>
+    </section>
+    <h2>
+      Introduction
+    </h2>
+    <p>
+      TBW
+    </p>
+    <h2>
+      Model
+    </h2>
+    <dl class="definitions">
+      <dt>
+        <dfn>digital identity credential</dfn>
+      </dt>
+      <dd>
+        A digital identity credential is a digital representation of a physical
+        identity credential (e.g., a drivers license, passport, etc). A digital
+        identity credential represents a set of claims about the subject of the
+        credential (e.g., the holder of the credential). A digital identity
+        credential is issued by an issuer (e.g., a government agency).
+      </dd>
+      <dt>
+        <dfn>Identity credential provider</dfn>
+      </dt>
+      <dd>
+        <p>
+          A credential provider is a an application or OS-level service that
+          presents UI to the user to request a digital identity credentia For
+          example,s a credential provider could be a mobile wallet application
+          that requests a mobile drivers license from the user.
+        </p>
+        <p>
+          A credential provider is responsible for presenting the user with a
+          list of [=digital credential/request protocol=] specific providers
+          (e.g., a list of mobile drivers license or applications that hold
+          driver's licenses).
+        </p>
+      </dd>
+      <dt>
+        <dfn for="digital identity">request protocol</dfn>
+      </dt>
+      <dd>
+        The request protocol is a standardized format/structure to [=request
+        digital identity credentials=]. For example, a request protocol could
+        be a "mDoc" for mobile drivers licenses as per [[?ISO-18013-5]].
+      </dd>
+    </dl>
+    <h2>
+      Scope
+    </h2>
+    <p>
+      The following is in is scope for the this specification:
+    </p>
+    <ul>
+      <li>Requesting a [=digital identity credentials=].
+      </li>
+    </ul>
+    <p>
+      The following is out of scope
+    </p>
+    <ul>
+      <li>Creating, updating, deleting, or otherwise managing [=digital
+      identity credentials=].
+      </li>
+    </ul>
+    <h2>
+      Extensions to the `CredentialsContainer` interface
+    </h2>
+    <pre class="idl">
+    partial interface CredentialsContainer {
+      Promise&lt;DigitalCredential&gt; requestIdentity(IdentityRequestOptions options);
```

Comment:
```suggestion
      Promise&lt;DigitalCredential&gt; requestIdentity(DigitalCredentialRequestOptions options);
```

Maybe we should use the DigitalCredential prefix for these mas, because otherwise we are going to run very quickly into a collision with the Identity Credential maps defined [here](https://fedidcg.github.io/FedCM/). For example, the [IdentityCredentialRequestOptions](https://fedidcg.github.io/FedCM/#enumdef-identitycredentialrequestoptionscontext).

Reply:
Yeah, good idea.